server/src/pages/shotindex/server.js

```diff
@@ -18,7 +18,7 @@ app.get("/", function(req, res) {
   }
   getShots.then(_render)
     .catch((err) => {
-      res.type("txt").status(500).send("Error rendering page: " + err);
+      res.type("txt").status(500).send("Error rendering page: " + err); // todo l10n: shotIndexPageErrorRendering
```

Comment:
Communicating l10n ids between the server and the clients was our best practice recommendation in Firefox OS. I'll follow it here as well. This means there's no need to store translations nor negotiate languages on the server at all.